src/main/java/com/softserve/actent/repository/ChatRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.softserve.actent.repository;
+
+import com.softserve.actent.model.entity.Chat;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface ChatRepository extends JpaRepository<Chat, Long> {
+
+    Chat findOneById(Long id);
```

Comment:
No need in this method in repository, since we have standard solutions for finding by id.

Reply:
its your opinion, but in my hand i want to check it for myself
sorry if i did hurt your feelings(